runtime/compiler/build/toolcfg/gnu-xlc/common.mk

```diff
@@ -85,7 +85,13 @@ CX_FLAGS_DEBUG+=-g -qfullpath
 
 CX_DEFAULTOPT=-O3
 CX_OPTFLAG?=$(CX_DEFAULTOPT)
-CX_FLAGS_PROD+=$(CX_OPTFLAG) -qdebug=nscrep
+CX_FLAGS_PROD+=$(CX_OPTFLAG)
+
+ifneq (,$(findstring ppc64le,$(PLATFORM)))
+    CX_FLAGS_PROD+=-qdebug=nscrep
+else
```

Comment:
Will this 'else' apply to Linux PPC BE only ?

Reply:
correct